src/proto/grpc/testing/test.proto

```diff
@@ -85,8 +85,20 @@ service LoadBalancerStatsService {
       returns (LoadBalancerStatsResponse) {}
 }
 
+// A service used to obtain real-time stats for verifying LB behavior.
+service LoadBalancerRealTimeStatsService {
+  // Gets the real-time stats for RPCs sent by a test client.
+  rpc GetClientRealTimeStats(grpc.testing.Empty)
```

Comment:
Let's define request for this RPC and response for the other. We may need them later.

Reply:
Sure. Added.